src/allocator_thread.c

```diff
@@ -289,7 +289,8 @@ void at_init(void) {
 void at_close(void) {
 	PFUNC();
 	const int msg = ATM_EXIT;
-	write(req_pipefd[1], &msg, sizeof(int));
+	if(write(req_pipefd[1], &msg, sizeof(int)) == -1)
```

Comment:
'tis better to use (void) write(...) here, since we dont want error message output from here.